### PR TITLE
Add files via upload

### LINKED ISSUE #775: GFG problems
### Is your feature request related to a problem? Please describe.
I would like to add 27 GFG problems.

### Describe the solution you'd like
These problems are solve in cpp.

**Would you like to work on this feature?**  
- [x] Yes  
- [ ] No

### PR DESCRIPTION
**Pull request by KashishJuneja101003**
### Description
I have added 27 GFG problems along with their solutions in cpp.

Fixes #775 

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/gfg solutions.md [hunk 1]
```diff
@@ -0,0 +1,1098 @@
...
+            }
+        }
+        
+        return result;
+    }
+};
+```
+
+## 7. Parenthesis Checker
+**Description:** Given an expression string x. Examine whether the pairs and the orders of {,},(,),[,] are correct in exp. For example, the function should return 'true' for exp = [()]{}{[()()]()} and 'false' for exp = [(]). Note: The driver code prints "balanced" if function return true, otherwise it prints "not balanced".
```
Comment (ajay-dhangar) [inline docs/gfg solutions.md]: replace `{,},(,),[,]` with ``` `{,},(,),[,]` ```

Comment (ajay-dhangar) [reply docs/gfg solutions.md]: and `[()]{}{[()()]()}` with ``` `[()]{}{[()()]()}` ```

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions

**Comment by KashishJuneja101003** [comment]
Kindly add hacktoberfest-accepted label also.
Thank You!

**Review by ajay-dhangar — APPROVED** [review]
Done

**Pull request merged**